src/Microsoft.AspNetCore.Html.Abstractions/HtmlEncodedString.cs

```diff
@@ -17,17 +17,20 @@ public class HtmlEncodedString : IHtmlContent
         /// </summary>
         public static readonly IHtmlContent NewLine = new HtmlEncodedString(Environment.NewLine);
 
-        private readonly string _value;
-
         /// <summary>
         /// Creates a new <see cref="HtmlEncodedString"/>.
         /// </summary>
         /// <param name="value">The HTML encoded value.</param>
         public HtmlEncodedString(string value)
         {
-            _value = value;
+            Value = value;
         }
 
+        /// <summary>
+        /// Gets the HTML encoded value.
+        /// </summary>
+        public string Value { get; }
```

Comment:
I commented on this, but it looks like github buried it when I added the docs.

This is required for a super beneficial optimization in writing out attributes for taghelpers in the common case that the attribute is plain old HTML text in the razor source.


> does it need to be on IHtmlContent or one of the other interfaces?

This only makes sense where this a prexisting